lgn-messages/src/types/v1/groth16/keys.rs

```diff
@@ -1,71 +0,0 @@
-use std::fmt::Display;
-
-use object_store::path::Path;
-use serde_derive::Deserialize;
-use serde_derive::Serialize;
-
-use crate::types::v1::query::keys::KEYS_QUERIES_PREFIX;
-
-pub type QueryId = String;
-
-/// Where to store the Groth16 proof
-#[derive(Debug, Clone, PartialEq, PartialOrd, Ord, Eq, Hash, Serialize, Deserialize)]
-pub struct ProofKey(pub QueryId);
-
-impl Display for ProofKey {
-    fn fmt(
-        &self,
-        f: &mut std::fmt::Formatter<'_>,
-    ) -> std::fmt::Result {
-        let query_id = &self.0;
-        write!(f, "{KEYS_QUERIES_PREFIX}/{query_id}/groth16")
-    }
-}
```

Comment:
Some this was dead code, some of this is used by the QE to store data in S3. This has been moved to the QE.